deployment/aws-waf-security-automations.template

```diff
@@ -2158,17 +2157,17 @@ Resources:
     Condition: SNSEmail
     Properties:
       DisplayName: 'AWS WAF Security Automations IP Expiration Notification'
-      TopicName: !Join ['-', ['AWS-WAF-Security-Automations-IP-Expiration-Notification', !GetAtt CreateUniqueID.UUID]]

```

Comment:
This will create a new topic each stack update, does not feel like the right approach